.travis-ci.sh

```diff
@@ -15,6 +15,10 @@ elif [[ $TASK = 'htmlproofer-external' ]]; then
   bundle exec jekyll build
   bundle exec htmlproofer $HTMLPROOFER_OPTIONS --external_only
 elif [[ $TASK = 'eip-validator' ]]; then
+  if [[ $(find . -maxdepth 1 -name 'eip-*' | wc -l) -ne 1 ]]; then
```

Comment:
I don't particularly think this is a good check (it is really fragile and not specific), but it is a net improvement compared to today.

Reply:
Agreed, I would prefer to do as you recommended and scan for front-matter outside the `EIPS`. But for now this is a net improvement.